sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -53,71 +50,35 @@ add the direct dependency to your project as follows.
 
 ### Prerequisites
 - A [Java Development Kit (JDK)][jdk_link], version 8 or later.
-- [Azure Subscription][azure_subscription]
+- An [Azure Subscription][azure_subscription].
 - An existing [Azure Key Vault Managed HSM][azure_keyvault_mhsm]. If you need to create a Managed HSM, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-key-vault-mhsm-name>` with your own, unique names:
 
 ```bash
-az keyvault create --hsm-name <your-key-vault-mhsm-name> --resource-group <your-resource-group-name> --administrators <your-service-principal-object-id> --location <your-azure-location>
+az keyvault create --hsm-name "<your-key-vault-mhsm-name>" --resource-group "<your-resource-group-name>" --administrators "<your-service-principal-object-id>" --location "<your-azure-location>"
 ```
 
 ### Authenticate the client
-In order to interact with the Azure Key Vault service, you'll need to either create an instance of the [KeyVaultAccessControlClient](#create-an-access-control-client) or an instance of the class [KeyVaultBackupClient](#create-a-backup-client). You would need a **vault url**, which you may see as "DNS Name" in the portal, and **client secret credentials (client id, client secret, tenant id)** to instantiate a client object using the default `DefaultAzureCredential` examples shown in this document.
+In order to interact with the Azure Key Vault service, you will need to either create an instance of the [`KeyVaultAccessControlClient`](#create-an-access-control-client) or an instance of the class [`KeyVaultBackupClient`](#create-a-backup-client). You will need a **vault url**, which you may see as "DNS Name" in the portal, and **[a managed identity][managed_identity]** to instantiate a client object using the default `DefaultAzureCredential` examples shown in this document.
```

Comment:
Looks like we need to add a `[managed_identity]` link to the bottom of the file to fix this reference